File: app/src/main/java/com/laushkina/anastasia/psychosophy/view/basesAndFunctions/BasesAndFunctionsAdapter.java
package com.laushkina.anastasia.psychosophy.view.basesAndFunctions;

import android.app.Fragment;
import android.app.FragmentManager;
import android.support.v13.app.FragmentPagerAdapter;

public class BasesAndFunctionsAdapter extends FragmentPagerAdapter {

    public static final int BASES = 0;
    public static final int FUNCTIONS = 1;
    private static final int PAGE_AMOUNT = 2;

    BasesAndFunctionsAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        Fragment fragment;
        switch (position){
            case BASES: {
                fragment = new BasesFragment();
                break;
            }
            case FUNCTIONS: {
                fragment = new BasesAndFunctions_FunctionsFragment();
                break;
            }
            default:
                throw new RuntimeException("Cannot detect bases and fragments page");
        }

        return fragment;
    }

    @Override
    public int getCount() {
        return PAGE_AMOUNT;
    }
}

File: app/src/main/java/com/laushkina/anastasia/psychosophy/view/basesAndFunctions/BasesAndFunctionsAdapter.kt
package com.laushkina.anastasia.psychosophy.view.basesAndFunctions

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter


class BasesAndFunctionsAdapter(fm: FragmentManager): FragmentPagerAdapter(fm) {
    companion object {
        const val BASES = 0
        const val FUNCTIONS = 1
        private const val PAGE_AMOUNT = 2
    }

    override fun getItem(position: Int): Fragment {
        return when (position) {
            BASES -> {
                BasesFragment()
            }
            FUNCTIONS -> {
                FunctionsDecriptionFragment()
            }
            else -> throw RuntimeException("Cannot detect bases and fragments page")
        }
    }

    override fun getCount(): Int {
       return PAGE_AMOUNT
    }
}